bin/pycbc_banksim

```diff
@@ -336,12 +369,25 @@ with ctx:
         # If not set fall back on filter low-freq cutoff
         if f_lower < 0.000001:
             f_lower = options.filter_low_frequency_cutoff
+        if f_lower < options.filter_low_frequency_cutoff:
```

Comment:
I changed my mind. I want this to be a warning. Maybe someone has made a ZDHP template bank and just wants to run it on O1-like data. Then it would be correct to raise the lower-frequency-cutoff. See cynical comment above, if the user ignores warnings.